fairing/config.py

```diff
@@ -36,7 +37,8 @@
 
 deployer_map = {
     'job': Job,
-    'tfjob': TfJob
+    'tfjob': TfJob,
+    'gcp': GCPJob
```

Comment:
GCP the right name? Or CMLE?

Reply:
I used GCP to keep it general as possible as a managed deployment option. What are your thoughts?

Reply:
LGTM